### PR TITLE
add support for subscribing for e-mail newsletter to FE API

### PR DESCRIPTION
**Pull request by malyMiso**
| Q             | A
| ------------- | ---
|Description, reason for the PR| add support for subscribing for e-mail newsletter to FE API
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/frontend-api/src/Model/Mutation/Newsletter/NewsletterMutation.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
+     * @param \Overblog\GraphQLBundle\Validator\InputValidator $validator
+     * @return bool[]
+     */
+    public function newsletterSubscribe(Argument $argument, InputValidator $validator): array
+    {
+        $validator->validate();
+
+        $input = $argument['input'];
+        $this->newsletterFacade->addSubscribedEmail($input['email'], $this->domain->getId());
```
Comment (grossmannmartin) [inline packages/frontend-api/src/Model/Mutation/Newsletter/NewsletterMutation.php:46]: what about trimming data? it's not necessary?

**Review by grossmannmartin — APPROVED** [review]
Can you check the data trimming?

Besides, it's fine by me (reviewed, tested)

File: packages/frontend-api/src/Model/Mutation/Newsletter/NewsletterMutation.php (hunk 1 above)
Comment (malyMiso) [reply packages/frontend-api/src/Model/Mutation/Newsletter/NewsletterMutation.php]: Yep, you're right. Fixed.

**Review by grossmannmartin — APPROVED** [review]
Thanks!

**Review by sslt — APPROVED** [review]

**Pull request merged**